src/components/general/SideSheet/Modal/styles.less

```diff
@@ -0,0 +1,94 @@
+@import '~styles/mixins.less';
+@import '~styles/variables.less';
+@import '~styles/colors.less';
+
+.container {
+    top: 0; bottom: 0;
+    left: 0; right: 0;
+    position: fixed;
+    opacity: 0;
+    transition: opacity .2s ease;
+    background-color: rgba(0, 0, 0, .32);
+
+    .modalSideSheet {
+        position: fixed;
+        top: 0; bottom: 0;
+        right: 0;
+        width: .grid-unit(80px)[];
+        transform: translateX(640px);
+        background: var(--color-white);
+        overflow: hidden;
+        display: flex;
+        flex-direction: column;
+        pointer-events: all;
+        transition: transform .2s ease;
+
+    
+        @media (max-width: @mobile-max-width) {
+            width: 100%;
+            left: 0;
+            transition: transform .1s ease-out;
+        }
+    
+        .scrollContainer {
+            width: 100%;
+            max-height: 100%;
+            overflow: auto;
+            flex-grow: 1;
+    
+            .content {
+                height: auto;
+            }
+        }
+    
+        .content {
+            height: 100%;
+        }
+    
+        .header {
+            padding: .grid-unit(1px)[] .grid-unit(3px)[];
+            background: var(--color-white);
+            display: flex;
+
+            .closeButton {
+                margin: 0;
+            }
+            .headerContent {
+                display: flex;
+                justify-content: space-between;
+                width: 100%;
+
+                .headerTitle{
+                    font-size: var(--header-font-size);
+                    line-height: var(--header-line-height);
+                    padding-left: .grid-unit(1px)[];
+                    flex: 1;
+                }
+    
+                .headerIcons{
+                    align-self: flex-end;
+                    display: flex;
+                    flex: 0;
+                }
+            }
+        }
+
+        &.comfortable {
+            --header-font-size: calc(var(--grid-unit) * 3px);
```

Comment:
Are these the only changes between comfy and compact?

Reply:
I think so, but there is no drawings of compact mode so I'll have to discuss with Lotte

Reply:
Discussed with Lotte, these are the only changes between comf and comp